src/patronus/tracing/logger.py

```diff
@@ -1,91 +1,271 @@
-import os
+import functools
+import logging
+import typing
+import uuid
+from enum import Enum
 from time import time_ns
-from typing import Any, Optional
+from types import MappingProxyType
+from typing import Optional, Union
 
-from opentelemetry._logs import SeverityNumber, set_logger_provider, get_logger
+from opentelemetry._logs import SeverityNumber
 from opentelemetry.exporter.otlp.proto.grpc._log_exporter import OTLPLogExporter as OTLPLogExporterTCP
+from opentelemetry.sdk._logs import Logger as OTELLogger
 from opentelemetry.sdk._logs import LoggerProvider as OTELLoggerProvider
-from opentelemetry.sdk._logs._internal import LogRecord
+from opentelemetry.sdk._logs import LoggingHandler, LogRecord
+from opentelemetry.sdk._logs._internal import ConcurrentMultiLogRecordProcessor, SynchronousMultiLogRecordProcessor
 from opentelemetry.sdk._logs.export import BatchLogRecordProcessor
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.util.instrumentation import InstrumentationScope
 from opentelemetry.trace import get_current_span
-from opentelemetry.util.types import Attributes
+from opentelemetry.util.types import Attributes as OTeLAttributes
 
-BASE_OTEL_ENDPOINT = 'https://otel.patronus.ai:4317'
-PAT_LOG_TYPE = "patronus-sdk-tracing"
+from patronus.config import config
+from patronus.context_utils import ContextObject, ResourceMutex
+from patronus.tracing.attributes import Attributes, format_service_name
 
 
-class Logger:
+class LogTypes(str, Enum):
+    evaluation_data = "evaluation_data"
```

Comment:
I think we should also update log type for logs created on /evaluation API call. Currently it's patronus-evaluation